.github/workflows/validate.yml

```diff
@@ -49,44 +47,24 @@ jobs:
     runs-on: ubuntu-latest
     if: needs.detect-changes.outputs.tb_pulumi == 'true'
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Run Ruff syntax checks
-        uses: astral-sh/ruff-action@v3
-        with:
-          src: './tb_pulumi'
+      - name: Checkout (GitHub)
+        uses: actions/checkout@v4
 
       - name: Run Ruff linter
-        uses: astral-sh/ruff-action@v3
+        uses: devcontainers/ci@v0.3
         with:
-          src: './tb_pulumi'
-          args: 'format --check --diff'
-
+          cacheFrom: ghcr.io/aatchison/pulumi/cached-devcontainer
```

Comment:
Any instance of the devcontainer imageName and cacheFrom would change to this repo. Need auth to use ghcr.io for container artifact